src/contract.ts

```diff
@@ -69,8 +64,7 @@ export async function getContractAt(web3: Web3, name: string, build_dir: string,
 export async function deployContract(web3: Web3, network: string, name: string, args: any[], build_dir: string, trace: boolean, defaultOptions: SendOptions, sendOptions: SendOptions): Promise<{contract: Contract, receipt: TransactionReceipt}> {
   const contractBuild = await getContractBuild(name, build_dir, trace);
   const contractAbi = JSON.parse(contractBuild.abi);
```

Comment:
I don't think you need the contractAbi her anymore since getContract doing that work now